docs/judge/problem_format.md

```diff
@@ -75,7 +75,7 @@ A problem with many possible outputs (e.g. not a single possible answer, with sc
 A `checker` Python script must implement a function that is called by the judge:
 
 ```python
-def check(proc_output, judge_output, judge_input, point_value, submission_source, **kwargs)
+def check(proc_output, judge_output, judge_input, point_value, submission_source, **kwargs):
```

Comment:
`judge_input`, `point_value`, and `submission_source` are all part of `kwargs` now; it should be `def check(proc_output, judge_output, **kwargs):`